### ISSUE TITLE
Error on initialization of RStudio 2022.07.1 Build 554 for Windows

### ISSUE THREAD

**Issue by dshanisko** [rstudio/rstudio#11773]
### System details

    RStudio Edition : 
    RStudio Version :  2022.07.1 Build 554 for Windows
    OS Version      :  Windows 10 Pro Version	10.0.19043 Build 19043
    R Version       :  4.1.2

### Steps to reproduce the problem
Hold CTRL, double click RStudio to launch with R 4.1.2 32bit

### Describe the problem in detail
RStudio Generates Error on attempt to launch R 4.1.2 32bit (or 4.0.3, 3.6.3). Following same procedure with 64bit version has successful launch. Process works with RStudio 2022.02.3 Build 492.  See generated error.

### RStudio Version

RStudio 2022.07.1+554, "Spotted Wakerobin" (7872775e, 2022-07-22) for Windows

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

### Error message

[No error available]

### Process Output

The R session exited with code 1. 

Error output:

```
[53232:25032:20220816,173722.888:ERROR crashpad_client_win.cc:491] CreateProcess: The system cannot find the file specified. (2)

```

Standard output:

```
[No output emitted]
```

### Logs

*C:/Users/dsh/AppData/Local/RStudio/log/rsession-dsh.log*

```
2022-08-16T22:37:22.888937Z [rsession-dsh] ERROR system error 22 (The device does not recognize the command); OCCURRED AT class rstudio::core::Error __cdecl rstudio::core::crash_handler::initialize(enum rstudio::core::crash_handler::ProgramMode) src/cpp/core/CrashHandler.cpp:358; LOGGED FROM: int __cdecl main(int,char *const []) C:\Users\jenkins\workspace\ide\open-source-pipeline\spotted-wakerobin\src\cpp\session\SessionMain.cpp:2007
2022-08-16T22:37:22.893889Z [rsession-dsh] ERROR system error 22 (The device does not recognize the command); OCCURRED AT class rstudio::core::Error __cdecl rstudio::core::crash_handler::initialize(enum rstudio::core::crash_handler::ProgramMode) src/cpp/core/CrashHandler.cpp:358; LOGGED FROM: int __cdecl main(int,char *const []) C:\Users\jenkins\workspace\ide\open-source-pipeline\spotted-wakerobin\src\cpp\session\SessionMain.cpp:2057
2022-08-16T22:37:22.893889Z [rsession-dsh] ERROR system error 22 (The device does not recognize the command); OCCURRED AT class rstudio::core::Error __cdecl rstudio::core::crash_handler::initialize(enum rstudio::core::crash_handler::ProgramMode) src/cpp/core/CrashHandler.cpp:358; LOGGED FROM: int __cdecl main(int,char *const []) C:\Users\jenkins\workspace\ide\open-source-pipeline\spotted-wakerobin\src\cpp\session\SessionMain.cpp:2057

```
                

### Describe the behavior you expected
Running Version of 32bit R 4.1.2

- [x ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jeffvroom** [comment]
I think this is because of the regression where crashhandler errors are now causing the session to exit.  See my last comment here:  https://github.com/rstudio/rstudio/issues/11717 but maybe disabling the crashhandler would be a workaround?

**Comment by David-S-Hanisko-NOAA** [comment]
Disabling crashhandler (unchecking "Send automated crash reports to RStudio") does allow the R 4.1.2 (4.0.3 and 3.6.3) 32bit versions to launch without producing the error report.

**Comment by ronblum** [comment]
Not sure if this is technically a duplicate or related, so I'll leave this open as a separate but related regression.

**Comment by MariaSemple** [comment]
We should check whether this is resolved once the change from #11717 is built.

**Comment by jonvanausdeln** [comment]
Verified on 2022.11.0-173
Windows11 (R v 4.1.2)

IDE starts up without issue selecting 32-bit R
